enhancements/security/bmc-ca-certificate-support.md

```diff
@@ -0,0 +1,294 @@
+---
+title: bmc-ca-certificate-support
+authors:
+  - "@Hellcatlk"
+  - "@zhouhao3"
+  - "@fenggw-fnst"
+reviewers:
+  - "@dtantsur"
+approvers:
+  - "@dtantsur"
+api-approvers:
+  - "@dtantsur"
+tracking-link:
+  - "https://issues.redhat.com/browse/RFE-3505"
+creation-date: 2023-04-13
+last-updated: 2023-05-22
+see-also:
+  - "/enhancements/this-other-neat-thing.md"
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# BMC CA Certificate Support
+
+## Summary
+
+This enhancement allows users to provide local CA certificates for IPI
+installation or subsequent node management, ensuring secure communication
+with baremetal BMCs that use certificates issued by the local CA or use
+self-signed certificates.
+
+## Motivation
+
+Currently, when deploying OCP clusters on baremetal using IPI or managing nodes
+on an existing cluster, users are unable to specify CA certificates. They can
+either use the system's default CA bundle for validation (which means they
+need to install certificates signed by a trusted CA for their BMCs) or disable
+certificate validation by setting disableCertificateVerification to false in
+BareMetalHost. This poses a security risk, particularly in production
+environments where such risks are unacceptable.
+
+### User Stories
+
+- As a cluster creator, I want to provide local CA certificates for IPI
+  installation, so that I can ensure secure communication with baremetal
+  BMCs using certificates issued by the local CA.
+
+- As a cluster creator, I want to utilize self-signed certificates for
+  baremetal BMCs during the IPI installation process, so that I can avoid
+  relying on external CAs for secure communication.
+
+- As a cluster administrator, I want to add new nodes to an existing cluster
+  securely using local-CA-signed certificates or self-signed certificates, so
+  that I can maintain a high level of security within the cluster.
+
+- As a cluster administrator, I want to update the CA certificates to address
+  various situations, such as renewing the validity of self-signed certificates
+  or trusting additional certificates issued by other local CAs, so that I can
+  ensure continued secure communication with BMCs.
+
+### Goals
+
+- Users provide CA certificates for IPI installation or node management after
```

Comment:
nit: "Users" is misleading here, the certificates will be provided by an administrator.